src/System.Management.Automation/engine/parser/token.cs

```diff
@@ -1142,17 +1146,20 @@ public static class TokenTraits
             /*              Command */ "command",
             /*               Hidden */ "hidden",
             /*                 Base */ "base",
+            /*              Dispose */ "dispose",
```

Comment:
Is this supposed to be `cleanup` now? Both the string and the comments?

Reply:
Yes and yes, will fix. 😁